src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -4278,10 +4589,17 @@ import Color from 'color';
       x,
       y,
       id: 'col_label',
+      opacity: is_gene_symbol_column
```

Comment:
This could just be `opacity: + !is_gene_symbol_column`, which gives you the numeric value of the (inverted) boolean.

Reply:
would that result in `0 or 1` or `0 or 0.3` ?

Reply:
it would result in 0 or 1 😃

Reply:
👀